dtbase/backend/api/sensor/routes.py

```diff
@@ -97,3 +161,95 @@ def list_sensors_of_a_type(type_name):
     """List sensors of a particular type in the database."""
     result = sensors.list_sensors(type_name=type_name, session=db.session)
     return jsonify(result), 200
+
+
+@blueprint.route("/list_sensor_types", methods=["GET"])
+# @login_required
+def list_sensor_types():
+    """List sensor types in the database."""
+    result = sensors.list_sensor_types(session=db.session)
+    return jsonify(result), 200
+
+
+@blueprint.route("/list_measures", methods=["GET"])
+# @login_required
+def list_sensor_measures():
+    """List sensor measures in the database."""
+    result = sensors.list_sensor_measures(session=db.session)
+    return jsonify(result), 200
+
+
+@blueprint.route("/sensor_readings", methods=["POST"])
```

Comment:
I feel like this should be a "GET" request rather than "POST", since we are retrieving information rather than storing it.    (We can still add a payload to a GET request, as is done here, or we could add it to the URL, I don't have a strong preference for either, so this is fine...)

Reply:
Ok! Changed to "GET", thanks!